Incremented version to 1.3.2.

gaiksaya opened the pull request:
Coming from https://github.com/opensearch-project/opensearch-build/issues/1805 post release activity.

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/performance-analyzer-rca/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

sruti1312 commented:
@gaiksaya  Build is failing because of the ```test-retry``` dependency version missing. I think it was made unintentionally during replacing 1.3.1 to 1.3.2. 

```
Plugin [id: 'org.gradle.test-retry', version: '1.3.2'] was not found in any of the following sources:

- Gradle Core Plugins (not a core plugin, please see https://docs.gradle.org/6.6.1/userguide/standard_plugins.html for available core plugins)
- Plugin Repositories (could not resolve plugin artifact 'org.gradle.test-retry:org.gradle.test-retry.gradle.plugin:1.3.2')
  Searched in the following repositories:
    Gradle Central Plugin Repository
```

gaiksaya commented:
> @gaiksaya Build is failing because of the `test-retry` dependency version missing. I think it was made unintentionally during replacing 1.3.1 to 1.3.2.
> 
> ```
> Plugin [id: 'org.gradle.test-retry', version: '1.3.2'] was not found in any of the following sources:
> 
> - Gradle Core Plugins (not a core plugin, please see https://docs.gradle.org/6.6.1/userguide/standard_plugins.html for available core plugins)
> - Plugin Repositories (could not resolve plugin artifact 'org.gradle.test-retry:org.gradle.test-retry.gradle.plugin:1.3.2')
>   Searched in the following repositories:
>     Gradle Central Plugin Repository
> ```

Fixed it. Also I removed the hard corded value for `opensearch.version` from workflows since default points to `1.3.2-SNAPSHOT`
